sdcm/remote/base.py

```diff
@@ -137,17 +137,18 @@ def _print_command_results(self, result: Result, verbose: bool, ignore_status: b
         """When verbose=True and ignore_status=True that means nothing will be printed in any case"""
         if verbose and not result.failed:
             if result.stderr:
```

Comment:
Please, define some local var for the following reasons:
- Better readability
- Reduce CPU actions by getting attr again and again (not critical, but why not?)

Example:
```
hostname = self.hostname
...
self.log.debug('<%s>: STDERR: %s', hostname, result.stderr)
...
self.log.debug('<%s>: Command "%s" finished with status %s', hostname, result.command, result.exited)
```

Reply:
ok, done